tooling/create-tauri-app/src/recipes/cljs.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019-2021 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+// import { join } from 'path'
+import { shell } from '../shell'
+import { Recipe } from '../types/recipe'
+
+const cljs: Recipe = {
+  descriptiveName: {
+    name: 'ClojureScript (https://clojurescript.org/)',
```

Comment:
The url in the name should link directly to the template site/github repo, check `svelte.ts` for example.